iroh-net/src/relay/client.rs

```diff
@@ -399,13 +396,13 @@ impl Client {
         }
     }
 
-    /// Connect to a relay Server and returns the underlying relay Client.
+    /// Connect to a relay Server and returns the underlying relay connection.
```

Comment:
now that we are here and the line is flagged as changed, can we pick one from imperative and third person? we have here both `Connect to` and `returns`, otherwise the change is an improvement imo